sihl/http/service.ml

```diff
@@ -0,0 +1,101 @@
+type meth =
+  | Get
+  | Post
+  | Put
+  | Delete
+  | Any
+
+type route = meth * string * (Opium_kernel.Request.t -> Opium_kernel.Response.t Lwt.t)
+
+type router =
+  { scope : string
+  ; routes : route list
+  ; middlewares : Opium_kernel.Rock.Middleware.t list
+  }
+
+let run_forever () =
+  let p, _ = Lwt.wait () in
+  p
+;;
+
+let prefix_route prefix (meth, path, handler) =
+  let path =
+    String.split_on_char '/' path
+    |> List.filter (fun e -> e != "")
```

Comment:
Only filter left empty string, not right

Reply:
Fixed.